### ISSUE TITLE
Links to logged-in user's profile + edit profile should be bigger (a11y)

### ISSUE THREAD

**Issue by clarfonthey** [mastodon/mastodon#3875]
These are rather small targets to click, especially on mobile, and it'd be nice if they could be made larger like the links on the Getting Started column.

**Comment by clarcharr** [comment]
Does anyone want to take on moving the edit profile link next to preferences? It shouldn't be too hard but I'm not exactly sure where to start for that.

**Comment by trwnh** [comment]
closing as stale, since the logged-out ui is no longer a separate thing
